radixdlt-java/src/test/java/com/radixdlt/client/application/identity/RadixIdentitiesTest.java

```diff
@@ -12,7 +12,7 @@
 	@Test
 	public void newEncryptedIdentityWriterTest() throws Exception {
 		Writer writer = mock(Writer.class);
-		RadixIdentities.createNewEncryptedIdentity(writer, "");
+		RadixIdentities.createNewEncryptedIdentity(writer, "Password");
```

Comment:
Interesting case - BouncyCastle do not allow empty password for key generation it failing with Empty Password exception. But if security provider is defined explicitly it will fail without fallback. I believe it is better to know which provider is used for key generation